_data/projects/ipfs.yml

```diff
@@ -0,0 +1,18 @@
+name: IPFS
+desc: "The InterPlanetary File System: The Permanent Web"
+site: https://ipfs.io
+tags:
+# Note those are tags categorizing your project, not issue labels.
+- *javascript*
```

Comment:
The YAML parser continues to complain - these entries shouldn't be wrapped in `*`s